projects/bitlayer-ybtc_b/index.js

```diff
@@ -1,26 +1,19 @@
 const { sumTokensExport, nullAddress } = require("../helper/sumTokens");
 
-const ybtc = '0x2cd3cdb3bd68eea0d3be81da707bc0c8743d7335'
+const ybtc_b = '0x2cd3cdb3bd68eea0d3be81da707bc0c8743d7335'
 
-// async function tvl(api) {
-//   const supply = await api.call({  abi: 'erc20:totalSupply', target: ybtc })
-//   api.add(ybtc, supply)
-// }
+async function tvl(api) {
+  const supply = await api.call({  abi: 'erc20:totalSupply', target: ybtc_b })
```

Comment:
please show us where the backing for the ybtc_b is

Reply:
avalanche ybtc_b is bridged from bitlayer chain via CCIP, bitlayer ybtc_b is minted from bitlayer native btc, here is the mint contract with balance data
https://www.btrscan.com/address/0x2cd3cdb3bd68eea0d3be81da707bc0c8743d7335?tab=Transactions


![img_v3_02ns_a5e642a8-ac9e-4048-a7e0-b8f47f382dhu](https://github.com/user-attachments/assets/a2b58efb-0280-42e4-a513-2c64ac794b07)